engines/gob/gameidtotype.h

```diff
@@ -69,6 +69,7 @@ static const GameIdToType gameIdToType[] = {
 	{                  "bambou", kGameTypeBambou },
 	{             "fascination", kGameTypeFascination },
 	{                  "geisha", kGameTypeGeisha },
+	{                    "adi1", kGameTypeNone },
```

Comment:
```suggestion
	{                    "adi1", kGameTypeAdi1 },
```
Also, please add kGameTypeAdi1 to the list of game types

Reply:
@bluegr done